_data/simple-icons.json

```diff
@@ -4411,6 +4411,11 @@
             "source": "https://dovetail.com/help/press-kit",
             "guidelines": "https://dovetail.com/help/press-kit"
         },
+        {
+            "title": "Downdetector",
+            "hex": "FF160A",
+            "source": "https://cdn2.downdetector.com/109e92fe03bc678/images/v2/Downdetector_reversecolor.svg"
```

Comment:
Please see [our guidelines on `source` links](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines). This should not be a link directly to an SVG file.